app/code/Magento/Downloadable/Setup/Patch/Data/InstallDownloadableAttributes.php

```diff
@@ -142,7 +142,7 @@ public function apply()
                 'visible' => false,
                 'required' => false,
                 'user_defined' => false,
-                'default' => '0',
+                'default' => null,
```

Comment:
@gelanivishal Thank you but that's not correct since this change won't apply to already installed instances of Magento. You need to create another named patch.